### PR TITLE
update changelog for release - Storage - Beta10

### PR DESCRIPTION
**Pull request by vhvb1989**

### CODE REVIEW

**Review by Jinming-Hu — APPROVED** [review]

File: sdk/storage/azure-storage-blobs/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 12.0.0-beta.10 (Unreleased)
+## 12.0.0-beta.10 (2021-03-30)
```
Comment (RickWinter) [inline sdk/storage/azure-storage-blobs/CHANGELOG.md:3]:
```suggestion
## 12.0.0-beta.10 (2021-04-06)
```

File: sdk/storage/azure-storage-common/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 12.0.0-beta.10 (Unreleased)
+## 12.0.0-beta.10 (2021-03-30)
```
Comment (RickWinter) [inline sdk/storage/azure-storage-common/CHANGELOG.md:3]:
```suggestion
## 12.0.0-beta.10 (2021-04-06)
```

File: sdk/storage/azure-storage-files-datalake/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 12.0.0-beta.10 (Unreleased)
+## 12.0.0-beta.10 (2021-03-30)
```
Comment (RickWinter) [inline sdk/storage/azure-storage-files-datalake/CHANGELOG.md:3]:
```suggestion
## 12.0.0-beta.10 (2021-04-06)
```

File: sdk/storage/azure-storage-files-shares/CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 12.0.0-beta.10 (Unreleased)
+## 12.0.0-beta.10 (2021-03-30)
```
Comment (RickWinter) [inline sdk/storage/azure-storage-files-shares/CHANGELOG.md:3]:
```suggestion
## 12.0.0-beta.10 (2021-04-06)
```

**Review by RickWinter — CHANGES_REQUESTED** [review]
Might be best to abandon this change let Storage make it when they release next week.

**Pull request closed without merging**